eng/dockerfile-templates/aspnet/Dockerfile

```diff
@@ -23,7 +23,16 @@ RUN curl -fSLo microsoft.net.compilers.2.9.0.zip https://api.nuget.org/packages/
     && %windir%\Microsoft.NET\Framework64\v4.0.30319\ngen install C:\RoslynCompilers\tools\vbc.exe /ExeConfig:C:\RoslynCompilers\tools\vbc.exe `
     && %windir%\Microsoft.NET\Framework64\v4.0.30319\ngen install C:\RoslynCompilers\tools\VBCSCompiler.exe /ExeConfig:C:\RoslynCompilers\tools\VBCSCompiler.exe
 
-ENV ROSLYN_COMPILER_LOCATION=C:\RoslynCompilers\tools
+# Install 3.6.0 Roslyn compilers
+RUN curl -fSLo microsoft.net.compilers.3.6.0.zip https://api.nuget.org/packages/microsoft.net.compilers.3.6.0.nupkg `
```

Comment:
This package is deprecated, please use microsoft.net.compilers.toolset instead 

From the NuPkg:

> Note: This package is deprecated. Please use Microsoft.Net.Compilers.Toolset instead

Reply:
Fine for this PR, but in the future we will stop producing new versions of this package.